.github/workflows/tekton_task_tests.yaml

```diff
@@ -57,3 +57,6 @@ jobs:
         run: .github/scripts/test_tekton_tasks.sh
         env:
           TASK_DIRS: ${{ steps.changed-dirs.outputs.all_changed_files }}
+          TASK_UPDATE_INFRA_DEPLOYMENT_PULL_SECRET: ${{ secrets.UPDATE_INFRA_DEPLOYMENT_PULL_SECRET }}
```

Comment:
I am fine with this. @davidmogar @mmalina are you?

Reply:
More over, I don't think it will work. The CI check failed:

```
secret "registry-io-pull-secret" not found
error: No valid secrets found
```

That's because secrets are not available for Pull Request events in Github Workflows (it's a security feature).

Reply:
> Due to the dangers inherent to automatic processing of PRs, GitHub’s standard pull_request workflow trigger by > default prevents write permissions and secrets access to the target repository. 

https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

Reply:
the workflow is not merged therefore the secrets are not injected. You can see from this PR run in my fork that it works:

https://github.com/scoheb/release-bundles/actions/runs/5836874507/job/15831323218?pr=2

Reply:
Hmm, I'm not sure how it worked - that seems to contradict what they write in the docs about not supplying secrets for the pull request event.

But I don't think it's because it's not merged yet. It's more likely related to using vs not using a fork. I found [this](https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow):

> With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a > forked repository.

Reply:
But my workflow run **is** from a forked repo.

Reply:
Unless I'm missing something, the one that worked is from a PR in your own fork - both origin and target branches are in the same repo: https://github.com/scoheb/release-bundles/pull/2 (And I assume you have that secret defined in your repo.)

Reply:
removed tests for now. resolving